README.md

```diff
@@ -20,7 +20,7 @@ comprised of the following components and features:
 
 # Usage
 
-    juju deploy cs:~containers/observable-kubernetes
+    conjure-up battlemidget/bundle-observable-kubernetes
```

Comment:
Please don't remove the juju deploy stanza. 

I get the want to spread the reach of conjure-up, but it really is in addition to, not a replacement for.